tools/github_stats.py

```diff
@@ -177,7 +177,10 @@ def report(issues, show_urls=False):
         all_authors = check_output(author_cmd).splitlines()
         unique_authors = sorted(set(all_authors))
 
-        print("The following %i authors contributed %i commits." % (len(unique_authors), ncommits))
+        if (len(unique_authors) != 0):
```

Comment:
How about the following?

   if len(unique_authors) == 0:
       print("No commits during this period.")
   else: 
      ... do all the rest